src/test/unit/features/alt-click.test.ts

```
@@ -0,0 +1,303 @@
+          get: sandbox.stub()
+        })
+      }
+    };
+    
+    // Set up global mocks
+    (global as any).vscode = mockVSCode;
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+    delete (global as any).vscode;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)